content.js

```diff
@@ -81,7 +81,12 @@ function trigger() {
 	toggleFiles();
 }
 
+function adjustOtherButtons() {
+	document.getElementsByClassName('new-pull-request-btn')[0].textContent = 'New PR';
+}
+
 document.addEventListener('DOMContentLoaded', () => {
+	adjustOtherButtons();
```

Comment:
Go to a repo, click the "Pull Request" tab, then click the "Code" tab, and you'll see the new PR button says "New pull request" again.